### PR TITLE
Desktop version 

### PR DESCRIPTION
**Pull request by patrick-amponsah**
This pull request enhances the layout and responsiveness of the website by implementing the following changes:
Used media queries in a CSS file to make the desktop version look like the template in Figma (Desktop: >=768px).
Used Grid and Flexbox in order to lay out the elements on the page.
Ensured adherence to the design provided in the Figma templates, focusing on maintaining consistency with regards to fonts, colors, images, text, and margins.
These changes improve the overall appearance, layout, and responsiveness of the website.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -753,4 +753,63 @@ hr {
   .project-detail-contener {
     padding: 24px;
   }
```
Comment (AlphaNtihinduka) [inline style.css]: - [ ] So far you have a good design 👏 . However, we need to do some improvements. I suggest that you ensure that the languages list is vertically centered in all the desktop screen sizes. In addition, I suggest that the width of the card is responsive to all the screen sizes. If you notice, at `772px` the contents of the card cross the borders of the card. This reduces the UI interface of the application. Therefore I request to apply these changes to ensure that the UI of your portfolio is improved.

Your design 😎 
image

Original design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]

File: style.css [hunk 2]
```diff
@@ -702,14 +780,17 @@ hr {
   }
 
   .about-me {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
+    /* display: grid; */
+
+    /* grid-template-columns: 1fr 1fr; */
```
Comment (Tufahel) [inline style.css]: - [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

File: style.css [hunk 3]
```diff
@@ -323,26 +368,40 @@ hr {
...
+
 .programming-lang {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
+  justify-content: space-between;
   background-color: #f7f7f9;
-  margin-right: 15px;
+
+  /* margin-right: 15px; */
```
Comment (Tufahel) [inline style.css]: - [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

File: style.css [hunk 4]
```diff
@@ -28,6 +36,15 @@ h2 {
   margin: 0;
 }
 
+.header-2 {
+  font-size: 2.5rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 3.25rem;
+
+  /* 130% */
```
Comment (Tufahel) [inline style.css]: - [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

File: style.css [hunk 5]
```diff
@@ -1,12 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
 
+/*
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} */
```
Comment (Tufahel) [inline style.css]: - [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

File: style.css [hunk 6]
```diff
@@ -702,14 +780,17 @@ hr {
...
+    display: flex;
+    flex-direction: row;
   }
 
   .programming-lang {
-    display: flex;
+    /* display: flex;
     flex-direction: column;
-    max-width: 80px;
+    max-width: 80px; */
```
Comment (Tufahel) [inline style.css]: - [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @patrick-amponsah :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ GitHub flow is correct.
✔️ Readme is professional.

## Required Changes ♻️

- [ ] As suggested by the previous reviewer the `contents` of the `cards` are overriding the cards and getting out of the cards. The issue is still not solved. Kindly fix this issue.
- image
- I would suggest you can try to remove `width` from the card `project-card` class and check whether it can be solved or not. It's just a suggestion. You can try different ways to solve it.
- image

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (Johnadibe) [reply style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css (hunk 4 above)
Comment (Johnadibe) [reply style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css (hunk 3 above)
Comment (Johnadibe) [reply style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css [hunk 7]
```diff
@@ -323,26 +402,40 @@ hr {
...
-  margin-right: 15px;
+
+  /* margin-right: 15px; */
   padding: 10px 20px;
   border-radius: 8px;
   list-style: none;
-  padding-left: 10px;
-  width: 90%;
+
+  /* padding-left: 10px; */
```
Comment (Johnadibe) [inline style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css [hunk 8]
```diff
@@ -323,26 +402,40 @@ hr {
...
+  max-width: 3rem;
 }
 
 .skills-title p {
   font-size: 20px;
   font-weight: 500;
-  line-height: 24px;
+  margin: 0;
+
+  /* line-height: 24px; */
```
Comment (Johnadibe) [inline style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css (hunk 2 above)
Comment (Johnadibe) [reply style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

File: style.css (hunk 6 above)
Comment (Johnadibe) [reply style.css]: > - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @patrick-amponsah🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Good commit history ✔️
- Your project is documented professionally✔️
- Implemented some changes requested by the previous reviewer✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @patrick-amponsah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Project Requirements fulfilled. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**